src/plugins/cosmos/components/modals/GetStarted/views/LearnMore.tsx

```diff
@@ -0,0 +1,151 @@
+import { ArrowBackIcon } from '@chakra-ui/icons'
+import { Box, Flex } from '@chakra-ui/layout'
+import { Button, IconButton } from '@chakra-ui/react'
+import { useSteps } from 'chakra-ui-steps'
+import { DefiModalHeader } from 'plugins/cosmos/components/DefiModalHeader/DefiModalHeader'
+import { useHistory } from 'react-router-dom'
+import rewards from 'assets/rewards.svg'
+import risk from 'assets/risk.svg'
+import withdraw from 'assets/withdraw.svg'
+import { CarouselDots } from 'components/CarouselDots/CarouselDots'
+import { SlideTransition } from 'components/SlideTransition'
+import { Text } from 'components/Text'
+import { useModal } from 'context/ModalProvider/ModalProvider'
+
+const STEPS_LENGTH = 3
```

Comment:
Can we use `Object.keys(STEP_TO_ELEMENTS_MAPPING).length` to get this dynamically?

Reply:
Great idea. If we're going to move this out of cosmos and make it generic, there could definitely be a variable amount of steps.